cirrus-ci_artifacts/cirrus-ci_artifacts.py

```diff
@@ -58,109 +61,91 @@ def get_raw_taskinfo(gqlclient, build_id):
         }
     ''')
     query_vars = dict(build_id=build_id)
-    result = gqlclient.execute(query, variable_values=query_vars)
-    if "build" in result and result["build"]:
-        result = result["build"]
-        if "tasks" in result and len(result["tasks"]):
-            return result["tasks"]
-        else:
-            raise RuntimeError(f"No tasks found for build with id {build_id}")
-    else:
-        raise RuntimeError(f"No Cirrus-CI build found with id {build_id}")
-
-
-def art_to_url(tid, artifacts, repo, bucket):
-    """Given an list of artifacts from a task object, return tuple of names and urls"""
-    if "/" not in repo:
-        raise RuntimeError(f"Expecting slash sep. repo. owner and name: '{repo}'")
+    tasks = gqlclient.execute(query, variable_values=query_vars)
+    if "build" in tasks and tasks["build"]:
+        _ = tasks["build"]
```

Comment:
Style: While legal, underscore is usually a placeholder rather than a general variable.

Reply:
Okay, replaced.